src/commands/install.yml

```diff
@@ -11,8 +11,8 @@ parameters:
     type: string
     default: ""
     description: >
-      Specify the full version tag to install. To install the latest LTS version, set the version to `lts`.
-      If unspecified, the version listed in .nvmrc will be installed. If no .nvmrc file exists the latest (current) version of Node.js will be installed by default.
+      Specify the full version tag to install. To install the latest version, set the version to `latest`.
+      If unspecified, the version listed in .nvmrc will be installed. If no .nvmrc file exists the current LTS version of Node.js will be installed by default.
```

Comment:
> If no .nvmrc file exists the current LTS version of Node.js

This can be confusing as "current" and "lts" both indicate release series from Node.js. Having those two words next to each other like that might be confusing.

Reply:
Fixed. Using `active` instead of `current`